PROVISIONING.md

```diff
@@ -20,11 +21,10 @@ tbd... re-celling, faults...
 ## Manual Provisioning
 
 tbd...
+Do we need to cover manual methods in our scope? Many, many methods. 
```

Comment:
at list let's list some of the common methods that are currently used... I know RHEL team listed a couple somewhere...